istio-2-mixin/links.libsonnet

```diff
@@ -0,0 +1,21 @@
+local g = import './g.libsonnet';
```

Comment:
Looks like the links default to "Back to ..." I think we should keep this as just the name of the dashboard. Traditionally its what been done. The windows exporter linking system is a bit wonky since its focused a lot more on a drill down aspect, which is why you see those back to verbiage.